assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -178,6 +178,25 @@
 				padding: 4px;
 			}
 		}
+
+		.googlesitekit-header__children {
+			align-items: center;
+			display: flex;
+			gap: $grid-gap-phone;
+			justify-content: flex-end;
+
+			@media (min-width: $bp-desktop) {
+				gap: $grid-gap-desktop;
+			}
+
+			div > button > .mdc-button__label {
+				display: none;
+
+				@media ( min-width: $width-wpAdminBarTablet + 1 + px ) {
```

Comment:
We cannot do a `$bp-wpAdminBarTablet + 1px` since `$bp-wpAdminBarTablet` already includes the `px` string. We instead used `$width-wpAdminBarTablet` which is the raw value of `$bp-wpAdminBarTablet`.